### ISSUE TITLE
Duplicate results for GetUserMedia

### ISSUE THREAD

**Issue by ewwink** [a-legotin/InstaSharper#6]
I want to get media for 1 page only, but it do http request 3 times with different URLs (max_id) and the results are duplicated because taken from first request.

```
var currentUserMedia = _instaApi.GetUserMedia(username, 1);
if (currentUserMedia.Succeeded)
{
  Console.WriteLine("Media count: " + currentUserMedia.Value.Count);
}
// 54 results not 18
```

**Comment by ewwink** [comment]
dirty fast fix, in `InstaApi.cs` add
`mediaList.Pages+=2;`
before
`while (mediaResponse.MoreAvailable && mediaList.Pages < maxPages)`

**Comment by ewwink** [comment]
found why its duplicated, maybe `converter.Convert() `in `mediaList.AddRange(converter.Convert())` is not reinitialized ?

```
var mediaResponse = JsonConvert.DeserializeObject < InstaMediaListResponse > (json,
  new InstaMediaListDataConverter());
var converter = ConvertersFabric.GetMediaListConverter(mediaResponse);
var mediaList = converter.Convert();
var nextId = mediaResponse.NextMaxId;
while (mediaResponse.MoreAvailable && mediaList.Pages < maxPages)
{
  instaUri = UriCreator.GetMediaListWithMaxIdUri(user.Pk, nextId);
  var nextMedia = await GetUserMediaListWithMaxIdAsync(instaUri);
  if (!nextMedia.Succeeded)
    Result.Success($ "Not all pages was downloaded: {nextMedia.Info.Message}", mediaList);
  nextId = nextMedia.Value.NextMaxId;
  mediaList.AddRange(converter.Convert());
  mediaList.Pages++;
}
return Result.Success(mediaList);
```

temporary fix add lines below before `mediaList.AddRange(converter.Convert());`

```
InstaMediaListResponse newMediaResponse = nextMedia.Value;
converter = ConvertersFabric.GetMediaListConverter(newMediaResponse);
```

**Comment by ewwink** [comment]
Another duplicate results found in `GetUserFollowersAsync` because `followersResponse` is not updated
change from

```
nextFollowers = await GetUserFollowersWithMaxIdAsync(username, nextFollowers.Value.NextMaxId);
if (!nextFollowers.Succeeded)
  Result.Success($ "Not all pages was downloaded: {nextFollowers.Info.Message}", followers);
followers.AddRange(....
```
to
```
nextFollowers = await GetUserFollowersWithMaxIdAsync(username, nextFollowers.Value.NextMaxId);
if (!nextFollowers.Succeeded)
  Result.Success($ "Not all pages was downloaded: {nextFollowers.Info.Message}", followers);
followersResponse = nextFollowers.Value;
followers.AddRange(......
```

**Comment by a-legotin** [comment]
Hello @ewwink i will check that and revert

**Comment by a-legotin** [comment]
closed as resolved, will be available in 1.2.3
